src/mscorlib/shared/System/IO/FileStream.Unix.cs

```diff
@@ -122,7 +122,7 @@ private void InitFromHandle(SafeFileHandle handle)
         /// <param name="access">The FileAccess provided to the stream's constructor</param>
         /// <param name="options">The FileOptions provided to the stream's constructor</param>
         /// <returns>The flags value to be passed to the open system call.</returns>
-        private static Interop.Sys.OpenFlags PreOpenConfigurationFromOptions(FileMode mode, FileAccess access, FileOptions options)
+        private static Interop.Sys.OpenFlags PreOpenConfigurationFromOptions(FileMode mode, FileAccess access, FileShare share, FileOptions options)
```

Comment:
Nit: please add `share` to the XML comments, just for consistency with the rest.